cli/src/main/java/org/jboss/gm/cli/Main.java

```diff
@@ -227,6 +220,8 @@ private void executeGradle(OutputStream out, boolean quiet, List<String> current
             logger.info("Executing CLI {} on Gradle project {} with JVM args '{}' and arguments '{}'",
                     ManifestUtils.getManifestInformation(Main.class), target, jvmArgs, currentGradleArgs);
 
+            logger.debug("Environment variables: {}", envVars);
```

Comment:
This ends up (at least on my system) dumping a **large** amount of extra output - around 132231 characters. While I don't think this is suitable for trace level output would a compromise be to add
```
envVars.remove("LS_COLORS");
```
which reduces it on my shell down to 4186 characters?

It could be argued that PROPT/RPROMPT/PS1 could also be removed which has the side benefit of removing any strange control characters in case they cause an issue (which reduces down to 3946 on my system) : 
```
            envVars.keySet().removeIf(k -> k.matches( "^(LS_COLORS|RPROMPT|PROMPT|PS1)$" ));
```

Reply:
The logger is needed for the test. Otherwise, I would just remove it. I don't think there's a way to probe the environment programmatically. I don't even see a `BuildLauncher::getEnvironmentVariables`.

Reply:
Resolved by using `keySet()`.